proto/interchain_security/ccv/provider/v1/provider.proto

```diff
@@ -37,14 +37,38 @@ message ConsumerAdditionProposal {
     // will be responsible for starting their consumer chain validator node.
     google.protobuf.Timestamp spawn_time = 7
         [(gogoproto.stdtime) = true, (gogoproto.nullable) = false];
-    // Indicates whether the outstanding unbonding operations should be released
-    // in case of a channel time-outs. When set to true, a governance proposal
-    // on the provider chain would be necessary to release the locked funds.
-    bool lock_unbonding_on_timeout = 8;   
-  }
+  
+    // BlocksPerDistributionTransmission is the number of blocks between ibc-token-transfers from the consumer chain to the provider chain.
+    // Note that at this transmission event a fraction of the accumulated tokens are divided and sent consumer redistribution address.
+    int64 blocks_per_distribution_transmission = 8;
```

Comment:
nit: reorder the fields to be grouped together, i.e., 

- unbonding_period
- ccv_timeout_period
- transfer_timeout_period
- consumer_redistribution_fraction
- blocks_per_distribution_transmission
- historical_entries


Maybe also add them to a `ConsumerParams` struct.

Reply:
Reordered but did not add struct